updatecli/weekly.d/jenkinscontroller-agents.yaml

```diff
@@ -20,6 +20,12 @@ sources:
       repository: "packer-images"
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
+  getLatestInboundAllInOneContainerImage:
+    kind: dockerdigest
+    spec:
+      image: "jenkinsciinfra/jenkins-agent-ubuntu-20.04"
+      tag: "latest"
+      architecture: amd64
   getLatestInboundRubyContainerImage:
```

Comment:
Can you remove this block as cleanup (either here or on a subsequent PR)? The ruby image is not there anymore